src/unit_tests/wazuh_db/test_wdb_integrity.c

```diff
@@ -268,7 +268,7 @@ static void test_wdbi_delete_success(void **state)
 
 static void test_wdbi_update_attempt_wbs_null(void **state)
 {
-    expect_assert_failure(wdbi_update_attempt(NULL, 0, 1));
+    expect_assert_failure(wdbi_update_attempt(NULL, 0, 1, NULL));
```

Comment:
Shouldn't we use a dummy checksum value here? Idem for other UT using NULL.

Reply:
Fixed